packages/suite/src/components/wallet/AccountsMenu/AccountItem.tsx

```diff
@@ -163,30 +170,54 @@ export const AccountItem = forwardRef((props: AccountItemProps, ref: React.Ref<H
                     <Row>
                         <AccountName data-test={`${dataTestKey}/label`}>{accountLabel}</AccountName>
                     </Row>
-                    <Row>
-                        <Balance>
-                            <CoinBalance value={account.formattedBalance} symbol={account.symbol} />
-                        </Balance>
-                        {(account.networkType === 'ethereum' ||
-                            account.networkType === 'cardano') &&
-                            !!account.tokens?.length && (
-                                <TokensCount
-                                    count={account.tokens.length}
-                                    onClick={handleClickOnTokens}
+                    {isBalanceShown && (
+                        <>
+                            <Row>
+                                <Balance>
+                                    <CoinBalance
+                                        value={account.formattedBalance}
+                                        symbol={account.symbol}
+                                    />
+                                </Balance>
+                                {(account.networkType === 'ethereum' ||
```

Comment:
nit: I would always create constants for long rendering conditions

Reply:
changed in different PR https://github.com/trezor/trezor-suite/pull/6925/commits/9672395806bb4838b218d36b6225133a9b350a2b#diff-131997ab8e041ace7864b31a291613ee2745e09da12474e223d2394bde8a84e9R135